lib/project_types/script/messages/messages.rb

```diff
@@ -50,6 +50,21 @@ module Messages
 
           script_not_found_cause: "Couldn't find script %s for extension point %s",
 
+          service_failure_cause: "Upstream service did not respond as expected",
```

Comment:
I don't think this is an accurate statement given some of the causes of the exception that yield this message coming up. E.g., bootstrap failures will raise this error when there is no network activity with the bootstrapping command invocation.

(sidebar, I think the code could benefit from the name of exception being changed so it's clearer as to which cases it applies to, I wasn't on the PR that introduced it)